mica/report/tests/test_write_report.py

```diff
@@ -2,9 +2,20 @@
 import tempfile
 import os
 import shutil
+import pytest
 
 from .. import report
 
+HAS_SYBASE_ACCESS = False
```

Comment:
Just a nit that this should be in the `except` block.  Essentially you are doing an `if / else`, and it makes the code logic more clear.

Yes:
```
if something:
    x = True
else:
    x = False
```
No, because the `x = False` is not explicitly linked to `something` in the code syntax.
```
x = False
if something:
    x = True
```
I'm sure I have written code along the "No" style, but I worry more about code clarity these days.